### PR TITLE
readme: add google translate links to support other languages

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

- Add google translate links to support other languages
- Fix a dead link in the pr template

### What is the related PR or file link(s)?

Reference link(s):

### CODE REVIEW

**Comment by CharLotteiu** [comment]
@lilin90 PTAL.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!--Thanks for your contribution to TiDB Cloud documentation. See [CONTRIBUTING](https://github.com/pingcap/community/blob/master/CONTRIBUTING.md) before filing this pull request (PR).-->
+<!--Thanks for your contribution to TiDB Cloud documentation. See [CONTRIBUTING](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md) before filing this pull request (PR).-->
```
[the remark this answers is not in the document]
Comment (CharLotteiu) [reply .github/pull_request_template.md:1]: It looks like the [contributing guidelines](https://github.com/pingcap/community/tree/master/contributors) is wrote for developers as there are guidelines for how to set up the development environment and get a code review. Are there any other contributing files related to documentation?

[the remark this answers is not in the document]
Comment (lilin90) [reply .github/pull_request_template.md:1]: Then it's the one in pingcap/docs. We can use this section: https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#how-to-contribute

[the remark this answers is not in the document]
Comment (lilin90) [reply .github/pull_request_template.md]: @CharLotteiu I updated the link in my comment.

**Review by lilin90 — APPROVED** [review]
LGTM

**Review by sivagao — COMMENTED** [review]
i recommend we can change from http://translate.google.com into  https://translate.google.com

**Comment by CharLotteiu** [comment]
@sivagao PTAL again.

**Review by sivagao — APPROVED** [review]

**Comment by sivagao** [comment]
LGTM

**Pull request merged**